api/database/migrations/2015_08_27_060129_create_revision_table.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+/**
+ * The class is intentionally named in singular, to not get a duplicate
+ * collision with the class CreateRevisionsTable in venturecraft/revisionable.
+ */
+class CreateRevisionTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('revisions', function (Blueprint $table) {
+            $table->increments('id');
```

Comment:
@xiphiaz Nope, an incremental shouldn't be needed. It was used in the original package to sort out old entries to be removed, though I changed that to use the creation time instead, so now it's pretty much a waste of space.